iphone/Classes/TiUIProgressBar.m

```diff
@@ -21,13 +21,13 @@ -(void)initializeTiLayoutView
 }
 #endif
 
--(id)initWithStyle:(UIProgressViewStyle)style_
+-(id)initWithStyle:(UIProgressViewStyle)_style andMinimumValue:(CGFloat)_min maximumValue:(CGFloat)_max;
 {
 	if (self = [super initWithFrame:CGRectZero])
 	{
-		style = style_;
-		min = 0;
-		max = 1;
+		style = _style;
+		min = _min;
+		max = _max;
```

Comment:
Yeah, but that are proxy-methods which expect values from the type `id`. Downcasting that just to use a setter didn't make sense to me. For me it's common to use this kind of setters in this case.